lib/internal/util/inspect.js

```diff
@@ -109,15 +117,24 @@ const {
   StringPrototypeToLowerCase,
   StringPrototypeTrim,
   StringPrototypeValueOf,
+  // TODO for when Node supports SuppressedError
+  // SuppressedError,
+  // SuppressedErrorPrototype,
```

Comment:
```suggestion
```

Reply:
I committed your suggestion. But I should have first asked why? Do you think such comments do not add value?

Reply:
It does add some value, but IMO it adds more noise than necessary – whoever addresses the TODO should be able to figure out that they need to destructure it from the `primordials` object. If for TODO we were commenting the actual implementation across the whole codebase, it would become unmaintainable.